Gaijinizer patch/Map037.txt

```diff
@@ -409,17 +415,18 @@ me! If you somehow do not go through here ...!#MTLed
 あんな見た目でも、アイツは¨魔王¨なんだ！！
 > CONTEXT: Map037/events/4/pages/0/255/Dialogue/other2
 > CONTEXT: Map037/events/4/pages/1/296/Dialogue/other2
-moron!!It is decided not to force! !
-Even if it looks like that, the guy is 魔 魔 ̈! !#MTLed
+You can't win, you dumb bastard!!
+
+She doesn't look it but she's the demon king!!#Ron
 > END STRING
 
 > BEGIN STRING
 
 \\N[1]は体が固まって、ピクリとも動けなくなった！
 > CONTEXT: Map037/events/4/pages/0/260/Dialogue
 > CONTEXT: Map037/events/4/pages/1/301/Dialogue
-\\N[1]'s body got stiff, and * twitch * couldn't move
-either!#MTLed
+\\N[1]'s body has gone stiff, he can't move even
+an inch!#Ron
 > END STRING
```

Comment:
Flow:
>_**N[1]'s body goes stiff...**_
>
>**_He can't even move an inch!_**

Honestly, we can probably wrap the last line in a red emphasis tag if you want. (\\\C[2] before "He")